spec/unit/property/list_spec.rb

```diff
@@ -32,11 +32,11 @@
     end
 
     it "should return the passed in array values joined with the delimiter from is_to_s" do
-      expect(@property.is_to_s(["foo","bar"])).to eq("foo,bar")
+      expect(@property.is_to_s(['foo','bar'])).to eq("['foo', 'bar']")
     end
 
-    it "should be able to correctly convert ':absent' to a string" do
-      expect(@property.is_to_s(:absent)).to eq("absent")
```

Comment:
Please also note that all such words were quoted in successful events prior to this change. The only thing that this change does, is that it's now also quoted in the event message produced in case of failure.